app/Espo/Services/Record.php

```diff
@@ -2312,11 +2321,66 @@ public function massUpdate($data, array $params)
             }
         }
 
+        if(!empty($hasChunks)){
+            MassUpdate::updatePublicData($this->getEntityType(), [
+                "chunkId" => $chunkId,
+                "total" => $total,
+                "totalChunks" => count($chunks)
+            ]);
+        }
+
         return $this
             ->dispatchEvent('afterMassUpdate', new Event(['data' => $data, 'service' => $this, 'result' => ['count' => count($ids), 'ids' => $ids]]))
             ->getArgument('result');
     }
 
+    public function getMassUpdatedItemsCount(string $chunkId)
+    {
+        $queueItems = $this->getEntityManager()->getConnection()
```

Comment:
no. It is a bad solution. You need to set into publicData jobs IDs, because you need to find the jobs fastly. Searching via LIKE is a bad idea.